packages/webview_flutter/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -192,6 +206,29 @@ - (void)onLoadUrl:(FlutterMethodCall*)call result:(FlutterResult)result {
   }
 }
```

Comment:
The legacy URL-based method should, as much as possible, be refactored to use the same flow as the new request-based method, so we don't have independent codepaths.